benchmarks/TFLite/CMakeLists.txt

```diff
@@ -158,6 +158,88 @@ set(ANDROID_MALI_GPU_TRANSLATION_FLAGS
   "--iree-vulkan-target-triple=valhall-unknown-android11"
 )
 
+set(X86_64_CASCADELAKE_CPU_TRANSLATION_FLAGS
+  "--iree-input-type=tosa"
+  "--iree-llvm-target-cpu=cascadelake"
+  "--iree-llvm-target-triple=x86_64-unknown-linux-gnu"
+)
+
+# CPU, Dylib-Sync, x86_64, full-inference
+iree_benchmark_suite(
+  MODULES
+    "${DEEPLABV3_FP32_MODULE}"
+    "${MOBILESSD_FP32_MODULE}"
+    "${POSENET_FP32_MODULE}"
+    "${MOBILEBERT_FP32_MODULE}"
+    "${MOBILENET_V2_MODULE}"
+    "${MOBILENET_V3SMALL_MODULE}"
+
+  BENCHMARK_MODES
+    "full-inference,default-flags"
+  TARGET_BACKEND
+    "dylib-llvm-aot"
+  TARGET_ARCHITECTURE
+    "CPU-x86-64-CascadeLake"
+  TRANSLATION_FLAGS
+    ${X86_64_CASCADELAKE_CPU_TRANSLATION_FLAGS}
+  BENCHMARK_TOOL
+    iree-benchmark-module
+  DRIVER
+    "dylib-sync"
+)
+
+# CPU, Dylib, 1 thread, x86_64, full-inference
+iree_benchmark_suite(
+  MODULES
+    "${DEEPLABV3_FP32_MODULE}"
+    "${MOBILESSD_FP32_MODULE}"
+    "${POSENET_FP32_MODULE}"
+    "${MOBILEBERT_FP32_MODULE}"
+    "${MOBILENET_V2_MODULE}"
+    "${MOBILENET_V3SMALL_MODULE}"
+
+  BENCHMARK_MODES
+    "1-thread,full-inference,default-flags"
+  TARGET_BACKEND
+    "dylib-llvm-aot"
+  TARGET_ARCHITECTURE
+    "CPU-x86-64-CascadeLake"
+  TRANSLATION_FLAGS
+    ${X86_64_CASCADELAKE_CPU_TRANSLATION_FLAGS}
+  BENCHMARK_TOOL
+    iree-benchmark-module
+  DRIVER
+    "dylib"
+  RUNTIME_FLAGS
+    "--task_topology_group_count=1"
+)
+
+# CPU, Dylib, 4 threads, x86_64, full-inference
+iree_benchmark_suite(
+  MODULES
+    "${DEEPLABV3_FP32_MODULE}"
+    "${MOBILESSD_FP32_MODULE}"
+    "${POSENET_FP32_MODULE}"
+    "${MOBILEBERT_FP32_MODULE}"
+    "${MOBILENET_V2_MODULE}"
+    "${MOBILENET_V3SMALL_MODULE}"
+
+  BENCHMARK_MODES
+    "4-thread,full-inference,default-flags"
+  TARGET_BACKEND
+    "dylib-llvm-aot"
+  TARGET_ARCHITECTURE
+    "CPU-x86-64-CascadeLake"
+  TRANSLATION_FLAGS
+    ${X86_64_CASCADELAKE_CPU_TRANSLATION_FLAGS}
+  BENCHMARK_TOOL
+    iree-benchmark-module
+  DRIVER
+    "dylib"
+  RUNTIME_FLAGS
+    "--task_topology_group_count=4"
+)
+
 # CPU, Dylib-Sync, big/little-core, full-inference
```

Comment:
These comments will be pretty similar between platforms, so I think it's time to either

A) Qualify which platform the suite is for in the comment
B) Split this file into multiple sections with those big comment blocks
C) Split this file into multiple _files_ (`benchmarks/TFLite/Linux` or `benchmarks/Linux/TFLite`, etc.)

Reply:
I'm planning the details for new benchmark pipeline now and it might restructure these files, so I'd like to keep the current change simple. I pulled out the x86 benchmarks into linux-x86-64.cmake to reduce the size of this file.